### DISCUSSION TITLE
v0.4.0

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#890] [Versions & Releases]
# 🐸 v0.4.0

- Update multi-speaker training API.
- VCTK recipes for all the TTS models.
- Documentation for multi-speaker training.
- Pre-trained Ukrainian GlowTTS model from 👑 https://github.com/robinhad/ukrainian-tts
- Pre-trained FastPitch VCTK model
- Dataset downloaders for LJSpeech and VCTK under `TTS.utils.downloaders`
- Documentation reformatting.
- Trainer V2  and compact. updates in model implementations.
       
    This update makes the Trainer V2  responsible for only the training of a model. The rest is excluded from the trainer and they need to be done either in the model or before calling the trainer. 

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.4.0'>v0.4.0</a>.</em>

**Comment by skol101** [comment]
This is a huge release! Thanks you!

**Comment by tekinek** [comment]
Congrats to new version!

Why DDC is off in tacotron2-DDC recipe? it is what makes coqui tacotron so good I think.
https://github.com/coqui-ai/TTS/blob/33aa27e2d6a80d38a7ea8928ad14d032d2c53a40/recipes/vctk/tacotron2-DDC/train_tacotron2-ddc.py#L38

**Reply by erogol** [reply]
good point. It's a mistake as I was testing the recipe on a smaller VRAM

**Comment by egorsmkv** [comment]
Ukrainian  model yayyy :)
